modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/product/navigation/control/menu/AnalyticsReportsProductNavigationControlMenuEntry.java

```diff
@@ -167,17 +165,6 @@ public boolean includeIcon(
 		return true;
 	}
 
-	public boolean isPanelStateOpen(HttpServletRequest httpServletRequest) {
```

Comment:
Hey, @RenanUmV move these methods to the interface and add the default implement of them into the `BaseProductNavigationControlMenuEntry` class. Thx

Reply:
Apply this for the others methods you have extract as well